capa/v25.1.3/release.yaml

```
@@ -0,0 +1,125 @@
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  name: aws-25.1.3
+spec:
+  apps:
+  - name: aws-ebs-csi-driver
+    version: 2.30.1
+    dependsOn:
+    - cloud-provider-aws
+  - name: aws-ebs-csi-driver-servicemonitors
+    version: 0.1.0
+    dependsOn:
+    - cert-manager
+  - catalog: default
+    name: aws-nth-bundle
+    version: 1.1.1
```

```suggestion
  - name: aws-nth-bundle
    version: 1.1.1
```

`default` is already the default via CRD. Also: Are there any dependencies?